tests/main.cpp

```diff
@@ -0,0 +1,3 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#define DOCTEST_CONFIG_COLORS_NONE
+#include "doctest.h"
```

Comment:
is this file still necessary?

Reply:
As far as i know this provides the entry point for the test suite

Reply:
alright!